File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height, i, j, k, m;
    
    //Taking Height
    
    do
    {
        Height = get_int("Height: ");
    }
    while (Height < 1 || Height > 8);
    
    m = Height - 1;
    
    //Printing Pyramids
    
    for (i = 1; i <= Height; i++)
    {
        // Printing Left Pyramid
        
        for (k = 1; k <= m; k++)
        {
            printf(" ");
        }
        
        for (j = 1; j <= i; j++)
        {
            printf("#");
        }
        
        printf("  ");
        
        // Printing Right Pyramid
        
        for (j = 1; j <= i; j++)
        {
            printf("#");
        }
        
        printf("\n");
        m--;
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# Prompting for valid input
while True:
    n = get_int("Height: ")
    # Checking input range
    if n >= 1 and n <= 8:
        break
    
# Printing pyramid
for i in range(n):
    
    # Left pyramid
    # Printing spaces
    for j in range(n - i - 1):
        print(" ", end="")
   
    # Printing bricks
    for j in range(i + 1):
        print("#", end="")
        
    # Printing spaces between pyramids    
    print("  ", end="")
    
    # Right pyramid
    for j in range(i + 1):
        print("#", end="")
        
    print()